addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -382,31 +383,31 @@ def _import_xml_invoice(self, content, attachment):
                             tree, './/CedentePrestatore')))
 
                 # Numbering attributed by the transmitter. <1.1.2>
-                elements = tree.xpath('//ProgressivoInvio', namespaces=tree.nsmap)
+                elements = tree.xpath('//ProgressivoInvio', namespaces=nsmap)
                 if elements:
                     invoice_form.name = elements[0].text
 
-                elements = body_tree.xpath('.//DatiGeneraliDocumento//Numero', namespaces=body_tree.nsmap)
```

Comment:
Is `body_tree.nsmap` equal to `tree.nsmap`?

Reply:
nope, if tree.nsmap is the one reported above body_tree.nsmap became
{
None: '',
'xsd': 'http://www.w3.org/2001/XMLSchema',
'xsi':'http://www.w3.org/2001/XMLSchema-instance'
}

Reply:
So shouldn't we do:
```
body_nsmap = {k: v for k, v in body_tree.nsmap.items() if k is not None}
```
?

Reply:
the first exception is raised here https://github.com/odoo/odoo/blob/c9bba369d11deee0fc91b6a6eaf2ef0c1c7f721a/addons/l10n_it_edi/models/account_invoice.py#L319
so I guess something bad occur during xpath parsing due to the provided namespace

Reply:
This thing has been copied from Factur-X, but FatturaPA does not have the cac: prefix e.g. in every tag.  I wonder whether we can not just remove all the namespaces=

Reply:
seems to work like a charm, updating the PR...